modules/issue_tracker/ajax/EditIssue.php

```diff
@@ -388,6 +388,10 @@ function getComments($issueID)
 
     //looping by reference so can edit in place
     foreach ($unformattedComments as &$comment) {
+        $comment["dateAddedFormatted"] = date(
```

Comment:
I think you should do as much formatting as possible here, so the backend sends back already clean and correct data, that react component simply displays